tslint.json

```diff
@@ -132,7 +132,7 @@
       }
     ],
     "directive-selector": [true, "attribute", "app", "camelCase"],
-    "component-selector": [true, "element", "app", "kebab-case"],
+    "component-selector": [true, "element", "app", "kebab-case", "pmp"],
```

Comment:
is this necessary?

Reply:
Without that the `ng lint` throws an error **The selector should be prefixed by "app"**